README.md

```diff
@@ -210,6 +210,9 @@ Examples:
   etrace --file trace.etl --stats
   etrace --clr GC --event GC/Start --field PID,TID,Reason[12],Type
   etrace --kernel Process --event Process/Start --where ImageFileName=myapp
+  etrace --kernel Process --event --where ProcessName=myProcessName
+  etrace --kernel Process --event --where Thread/Start ProcessId=4
```

Comment:
@Pavel-Durov I don't think is the right order, is it? Also, why put `--event` there if you're not including any events to filter?